src/app/organizations/oss.module.ts

```diff
@@ -0,0 +1,283 @@
+import { DragDropModule } from "@angular/cdk/drag-drop";
```

Comment:
I believe you forgot to remove this file.

Reply:
Pushing code at 5pm on a Friday 😁